tests/unit/converter/ena/test_ena_run.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+
+from config import ENA_FTP_DIR
+from converter.ena.classes import FileFiletype
+from converter.ena.ena import HcaData
+from converter.ena.ena_run import EnaRun
+from xsdata.formats.dataclass.serializers import XmlSerializer
+from xsdata.formats.dataclass.serializers.config import SerializerConfig
+
+
+class TestEnaRun(unittest.TestCase):
+    serializer = XmlSerializer(config=SerializerConfig(pretty_print=True))
```

Comment:
Could a `RUN.XML` contain more than 1 run?
If yes, then it would be nice to cover that case.
Tx